docs/overrides/main.html

```diff
@@ -1,10 +1,10 @@
 {% extends "base.html" %}
 {% block announce %}
 <div style="color: white;">
-  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-269" style="color: #07e798;">
-    <strong>Prefect 2.6.9 is live!</strong>
-  </a> Logging into Prefect Cloud from the CLI has been given a serious upgrade! Plus more
-  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-269" style="color: #07e798;">
+  <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-270" style="color: #07e798;">
+    <strong>Prefect 2.7.0 is live!</strong>
+  </a> Cancel flow runs from the CLI, UI, REST API, or Python client! Plus more
```

Comment:
I'd do "Cancellation, pausing, logging prints, agent limits, and more!"

Reply:
I think that cancellation can happen from multiple locations is less exciting than the other features.

Reply:
"Cancel, pause, and resume flow runs, log prints, agent limits, plus more fixes & enhancements"

Reply:
Sure!